### PR TITLE
[TACHYON-1428] Documented parts of the tachyon-servers module in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: servers/src/test/java/tachyon/master/lineage/LineageMasterTest.java [hunk 1]
```diff
@@ -143,6 +183,11 @@ public void reinitializeFileTest() throws Exception {
     Mockito.verify(mFileSystemMaster).reinitializeFile(new TachyonURI("/test1"), 500L, 10L);
   }
 
+  /**
+   *
```
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/lineage/LineageMasterTest.java]: Could you add the doc here?

**Comment by yupeng9** [comment]
Thanks for the doc improvement! I left one comment.

**Comment by HorizonNet** [comment]
@yupeng9 Thanks for the comment. I updated the PR accordingly.

**Comment by yupeng9** [comment]
LGTM. Thanks!

**Pull request merged**